resources/js/Shared/Layout/SearchPanelSkeleton.vue

```diff
@@ -1,12 +1,15 @@
+<script setup>
+</script>
+
 <template>
```

Comment:
```suggestion
<template>
```
You can remove `<script>` when you don't use it.